framework/player/init/unitTracker.sqf

```diff
@@ -12,35 +13,39 @@ lmf_player_tracklist = [];
 
 // ADD PLAYERS CYCLE //////////////////////////////////////////////////////////////////////////////
 [] spawn {
-	private _fnc_addUnitMarker =  {
-		private _unit = _this select 0;
-		private _marker = createMarkerLocal ["nk_m_"+str(_unit),getposASL _unit];
-		_marker setMarkerShapeLocal "Icon";
-		_marker setMarkerColorLocal var_markerSide;
-		_marker setMarkerTypeLocal "mil_dot_noshadow";
-		_marker setmarkerSizeLocal [0.4,0.4];
+	private _fnc_addUnit =  {
+		private _unit = _this#0;
 		lmf_player_tracklist pushbackunique _unit;
 	};
+
 	while {true} do {
+		//ADD PLAYERS FROM PLAYER GROUP TO LIST (MINUS GROUP LEADER)
+		{
+			if (_x in lmf_player_tracklist) then {} else {0 =[_x] call _fnc_addUnit};
```

Comment:
findif and negate the condition so the function call is in the `then` block instead. Also `0= ... call ... ` is crap as your do not use your if statement to assign a variable.